tests/appsec/api_security/test_apisec_sampling.py

```diff
@@ -56,3 +57,107 @@ def test_sampling_rate(self):
         )
         logger.info(f"API SECURITY SAMPLING RESULT: diff is {diff:.2f} standard deviations")
         assert diff <= 4.0, "sampling rate is not 0.1"
+
+
+@rfc("https://docs.google.com/document/d/1PYoHms9PPXR8V_5_T5-KXAhoFDKQYA8mTnmS12xkGOE")
+@scenarios.appsec_api_security_with_sampling
+@features.api_security_configuration
+class Test_API_Security_Sampling_Different_Endpoints:
+    """Test API Security - with different endpoints"""
+
+    def setup_sampling_delay(self):
+        self.request1 = weblog.get("/api_security/sampling/200")
+        self.request2 = weblog.get("/sample_rate_route/1")
+        self.all_requests = [weblog.get("/api_security/sampling/200") for _ in range(10)]
+
+    def test_sampling_delay(self):
+
+        assert self.request1.status_code == 200
+        schema1 = get_schema(self.request1, "req.headers")
+        assert schema1 is not None
+
+        assert self.request2.status_code == 200
+        schema2 = get_schema(self.request2, "req.headers")
+        assert schema2 is not None
+
+        assert all(r.status_code == 200 for r in self.all_requests)
+        assert all(get_schema(r, "req.headers") is None for r in self.all_requests)
+
+
+@rfc("https://docs.google.com/document/d/1PYoHms9PPXR8V_5_T5-KXAhoFDKQYA8mTnmS12xkGOE")
+@scenarios.appsec_api_security_with_sampling
+@features.api_security_configuration
+class Test_API_Security_Sampling_Different_Paths:
+    """Test API Security - same endpoints but different paths"""
+
+    def setup_sampling_delay(self):
+        # Wait for 10s to avoid other tests calling same endpoints
+        time.sleep(10)
+        self.request1 = weblog.get("/sample_rate_route/11")
```

Comment:
Use a dedicated endpoint for the test + docs in `docs/weblog` making explicit that this endpoint  is reserved.

I let you choose the good prefix, I just put `api_security` because I'm lazy 

```suggestion
        self.request1 = weblog.get("/api_security/sample_rate_route/11")
```